driver/src/main/java/org/dcache/SapphireDriver.java

```diff
@@ -111,16 +321,28 @@ public void remove(Iterable<RemoveRequest> requests)
     @Override
     public void cancel(UUID uuid)
     {
-        _log.debug("Cancel triggered for UUID {}", uuid);
-        Predicate<FlushRequest> byUUID = request -> request.getId().equals(uuid);
-        flushRequestQueue.stream().filter(byUUID)
+        LOGGER.debug("Cancel triggered for UUID {}", uuid);
+        Predicate<FlushRequest> flushByUUID = request -> request.getId().equals(uuid);
```

Comment:
you `Predicate<NearlineRequest>`, then you need only one `filterByUUID`

Reply:
I don't know how I should do it or if I understood it correctly. If I use `Predicate<NearlineRequest>` I still have to `filterByUUID` twice as I have two queues where the request may be in. Beside, is it working or a good idea to use NearlineRequest without giving a type?

Reply:
I Think so. This is just a filter :man_shrugging: